src/pages/claims/[claimId].svelte

```diff
@@ -1,14 +1,24 @@
 <script lang="ts">
-import user from '../../authn/user'
-import { Banner, Breadcrumb, ClaimBanner, ConvertCurrencyLink, FileDropArea, FilePreview, MoneyInput, Row } from '../../components'
+import user from '../../authn/user.js'
```

Comment:
remove the extraneous '.js'
```suggestion
import user from '../../authn/user'
```